src/ThreadedSensitivities.jl

```diff
@@ -34,3 +34,20 @@ function generatesensitivityodes(react,sol)
     end
     return sa_list
 end
+
+"""
+Combine ODE solutions into a sensitivity solution
+"""
+function generatesenssolution(sol,sensdict,sensprob)
+    ts = sol.t
+    ordkeys = sort([x for x in keys(sensdict)])
```

Comment:
I feel the list comprehension is not needed, can we use `sort(list(keys(sensdict)))` or something like this?

Reply:
`Array` doesn't work at least. I'm not sure if there's a better more readable way than that.